usr/src/uts/armv8/rpi4/io/rpi4.c

```diff
@@ -69,7 +77,7 @@ plat_get_cpu_clock(int cpu_no)
 	if (node == 0)
 		cmn_err(CE_PANIC, "cprman register is not found");
 
-	struct prom_hwclock clk = { node, VCPROP_CLK_ARM };
+	struct prom_hwclock clk = { node, DTPROP_CLK_ARM };
 	err = plat_hwclock_get_rate(&clk);
 	if (err == -1)
 		return (1500 * 1000 * 1000);
```

Comment:
Since we can read the clock rate now, should we panic if we could not?

Reply:
Maybe? I don't have a well formed opinion here, so if anybody feels strongly that we should, then I'll make that change.

Reply:
I think panicking is fine, we're in machdep code where something machdep has gone really confusingly wrong.

you could also just use `CE_WARN` if you think panicking is heavy handed

Reply:
I've gone with `CE_PANIC` for now. If we run into trouble we can back it down to a `CE_WARN`.